src/utilities/FieldPlaneAveraging.cpp

```diff
@@ -0,0 +1,143 @@
+//
+//  FieldPlaneAveraging.cpp
+//  amr-wind
+
+
+#include "FieldPlaneAveraging.H"
+#include <algorithm>
+#include "incflo.H"
+
+using namespace amrex;
+
+
+Real FieldPlaneAveraging::eval_line_average(Real x, int comp) const
```

Comment:
For me this method name is ambiguous. It is not computing the averages on the line, instead it is interpolating a previously computed average at a given point along the line.